src/platform-integration/ios/app-extensions.md

```diff
@@ -228,9 +252,9 @@ Revisit these docs after future Xcode releases to see if they are fixed.
 
 1. Build and run the main application target.
 1. After the app is launched on the simulator,
-   press <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>H</kbd> to minimize the app,
-   which switches to the home screen.
-1. Launch an app that supports Share Extension,
+   press <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>H</kbd>
+   to minimize the app, which switches to the home screen.
+1. Launch an app that supports the Share Extension,
```

Comment:
**question:** Should Share Extension be capitalized throughout? If not, is it UI text? If that's the case, bold it?

Reply:
No, it shouldn't be Share Extension, but I missed a couple. It should be either "Share extension" or "share extension" depending on context.